fcl_product_components/components/gne_reference_pane/sass/gne-reference-pane.scss

```diff
@@ -0,0 +1,43 @@
+$include-gne-reference-styles: true !default;
+
+// Default component class.
+$gne-reference-pane-selector: '.pane-fcl-procat-product-gne-reference-pane';
+$gne-reference-selector: '.gne-reference-pane' !default;
+$gne-reference-pane-gne-reference-display: inline-block !default;
```

Comment:
For now these are fine, but we have started to avoid  including margin, padding, display props etc in variables. 

Things we are including:
```

- Colors
- Font properties (Font-size, Font-color etc)
- Line height
```